zombienet-tests/zkv-lib/index.js

```diff
@@ -166,7 +166,7 @@ async function waitForEvent(api, timeout, pallet, name) {
 
 exports.registerVk = async (pallet, signer, vk) => {
     return await submitExtrinsic(api, pallet.registerVk(vk), signer, BlockUntil.InBlock,
-        (event) => event.section == "settlementFFlonkPallet" && event.method == "VkRegistered"
+        (event) => event.section == "settlementUltraplonkPallet" && event.method == "VkRegistered"
```

Comment:
Maybe that's just a _bug_ ... it's working just for ultraplonk... but it's still fine